### ISSUE TITLE
ViewWizard.exe report virus alert

### ISSUE THREAD

**Issue by IronGator** [blackrosezy/gui-inspect-tool#1]
Threat Name:    GenericR-EDW!8F6676BC5C8C
Threat Type:    Trojan

**Comment by blackrosezy** [comment]
This is false positive. Please view the scan result at https://www.virustotal.com/en/file/8458597d7b110c07a928c893d2ce9e3348087c77fc07837250cdf13d2378cfa1/analysis/

`Kaspersky`, `BitDefender`, `ESET-NOD32` and `F-Secure` reported no virus found.

**Comment by IronGator** [comment]
Thanks for the review.

Just to let you know the same issues may happen to WSEdit.EXE

On McAfee:
Threat Name:    GenericR-FEQ!B2F3C7E137F4
Threat Type:    Trojan

**Comment by alexandruchirita4192** [comment]
There are too many antiviruses reporting these 2 "tools".

WSEdit.EXE has 14 "false-positives":
https://www.virustotal.com/gui/file/d62ba1f3f9579eddd49cd44dc8e03ea096d77867ece60883f7d55784b48f8c71/detection

and ViewWizard.exe has 22 "false-positives":
https://www.virustotal.com/gui/file/8458597d7b110c07a928c893d2ce9e3348087c77fc07837250cdf13d2378cfa1/detection

How stupid do you think antiviruses are, to be so many false-positives for these files from so many antiviruses at once?

You're certainly doing something weird in those "tools", at least..

**Comment by nm17** [comment]
```
ViewWizard.exe: Win.Trojan.Pef13c-2 FOUND
WSEdit.EXE: Win.Trojan.Agent-1218576 FOUND
```
